discovery-infra/test_infra/controllers/node_controllers/libvirt_controller.py

```diff
@@ -1,18 +1,24 @@
 import os
 import logging
 import libvirt
-from test_infra import consts
+import waiting
 from test_infra import utils
+from test_infra import consts
+from test_infra.controllers.node_controllers.host import Host
 from test_infra.controllers.node_controllers.node_controller import NodeController
 
 
 class LibvirtController(NodeController):
 
     def __init__(self, **kwargs):
         self.libvirt_connection = libvirt.open('qemu:///system')
+        self.private_ssh_key_path = kwargs.get("private_ssh_key_path")
 
     def __del__(self):
-        self.libvirt_connection.close()
+        try:
```

Comment:
Use `suppress` from `contextlib` and please provide specific Exceptions you.

Reply:
first of all it don't want to provide specific exception here.That's why i didn't.

Reply:
i mean anything, no matter what

Reply:
It's bad practice. For example if there would be a `RecursionError` or a `ImportError`, `NotImplementedError` you would want to know about it that you don't call the correct function.